File: lib/licensing/trust-anchors.ts
/**
 * Issuer public keys shipped with a product release.
 *
 * Production release automation inserts active public keys here. Do not load
 * trust anchors from environment variables, the database, or clinic-writable
 * files. Issuer private keys must never enter this repository or installation.
 */
export const ISSUER_PUBLIC_KEYS: Readonly<Record<string, string>> = Object.freeze({})

File: backend/app/licensing/trust_anchors.py
"""Issuer trust anchors shipped read-only with the product.

Public keys are intentionally code-owned rather than loaded from the clinic state
directory. Production release automation inserts active issuer public keys here.
Private keys never enter this repository or any clinic installation.
"""

from types import MappingProxyType

ISSUER_PUBLIC_KEYS: MappingProxyType[str, str] = MappingProxyType({})
